Alexa.NET.Tests/ResponseTests.cs

```diff
@@ -248,6 +248,24 @@ public void DeserializesExampleAskForPermissionsConsentJson()
             Assert.Equal("read::alexa:household:list", card.Permissions.First());
         }
 
+        [Fact]
+        public void SerializesPlainTextOutputSpeechToJson()
+        {
+            var plainTextOutputSpeech = new PlainTextOutputSpeech { Text = "text content" };
+            var json = JsonConvert.SerializeObject(plainTextOutputSpeech, Formatting.Indented, new JsonSerializerSettings
```

Comment:
After the speech output has been created, to ensure that all the tests use the same serialization, line 255-266 can become `Assert.True(Utility.CompareJson(plainTextOutputSpeech, "PlainTextOutputSpeech.json"));`

Reply:
@stoiveyp Done.